core/src/main/kotlin/com/malinskiy/marathon/cache/CacheEntryReader.kt

```diff
@@ -0,0 +1,7 @@
+package com.malinskiy.marathon.cache
+
+import java.io.InputStream
+
+interface CacheEntryReader {
+    fun readFrom(input: InputStream)
```

Comment:
Will it throw exception if there will be error when reading or maybe it should return boolean?

Reply:
Yes, it will throw an exception in case of error